File: Python/coins.py
def coins(dollars, cents):
    total = dollars*100 + cents
    quarters = total//25
    total = 0 if total <= 0 else total % 25
    dimes = total//10
    total = 0 if total <= 0 else total % 10
    nickels = total//5
    total = 0 if total <= 0 else total % 5
    pennies = total
    return quarters, dimes, nickels, pennies


if __name__ == "__main__":
    di = int(input("# of dollars: "))
    ci = int(input("# of cents: "))
    q, d, n, p = coins(di, ci)
    print(f"The coins are {q} quarters, {d} dimes, {n} nickels and {p} pennies")

File: C++/coins.cpp
#include <iostream>


int main() {
    int dollars, cents, total;
    int quarters, dimes, nickels, pennies;

    std::cout << "Please enter the amount of money to convert:" << std::endl;
    std::cout << "# of dollars: ";
    std::cin >> dollars;
    std::cout << "# of cents: ";
    std::cin >> cents;

    total = dollars*100 + cents;
    quarters = total/25;
    total = (total > 0 ? total % 25 : 0);
    dimes = total/10;
    total = (total > 0 ? total % 10 : 0);
    nickels = total/5;
    total = (total > 0 ? total % 5 : 0);
    pennies = total;

    std::cout << "The coins are "
              << quarters
              << " quarters, "
              << dimes
              << " dimes, "
              << nickels
              << " nickels and "
              << pennies
              << " pennies";

    return 0;
}
